frontend/src/components/App/Notifications/index.tsx

```diff
@@ -178,6 +178,10 @@ export default function Notifications() {
   const history = useHistory();
 
   useEffect(() => {
+    // only perform processing if the bell icon is clicked
+    if (!anchorEl) {
+      return;
+    }
```

Comment:
AFAIU this still affects the normal functioning of the notifications. (or maybe you left this here by mistake)